### PR TITLE
Genre

### PR DESCRIPTION
**Pull request by Fayob**
- Add genre class
- Add genre properties 
- Implement genre method

### CODE REVIEW

File: src/item.rb
Comment (Elerqsousy) [inline src/item.rb:12]: Removing the `archived` default value will result in making it a mandatory attr for initialization. 
Instead leaving it optional will help initiate the class with fewer arguments and support the original work flow.
```suggestion
  def initialize(publish_date, archived: false)
    @id = rand(100...1000)
    @publish_date = publish_date
    @archived = archived
  end
```

**Review by Elerqsousy — CHANGES_REQUESTED** [review]
Great job so far.
There is a conflict with the dev branch. Kindly, resolve the conflict first before attempting to merge. I have left some requested changes as well.
Feel free to get back to me if there are any questions or concerns

File: src/item.rb
Comment (Fayob) [reply src/item.rb]: It's not a conflict, it seems it requires an approval before merging

**Review by Elerqsousy — APPROVED** [review]
Perfect job, and I am sorry for the confusion.
Please feel free to merge.

**Pull request merged**
